Log Entry : 12:19:58, martes, 6 de junio de 2017 :: From GenericRep8 :: An error occurred while executing the command definition. See the inner exception for details.
 :Inner Exception: System.InvalidOperationException: Ya hay un DataReader abierto asociado a este Command, debe cerrarlo primero.
   en System.Data.SqlClient.SqlInternalConnectionTds.ValidateConnectionForExecute(SqlCommand command)
   en System.Data.SqlClient.SqlConnection.ValidateConnectionForExecute(String method, SqlCommand command)
   en System.Data.SqlClient.SqlCommand.ValidateCommand(String method, Boolean async)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   en System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   en System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   en System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   en System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
------------------------------------------------------------
Log Entry : 12:38:13, martes, 6 de junio de 2017 :: From GenericRep8 :: An error occurred while executing the command definition. See the inner exception for details.
 :Inner Exception: System.InvalidOperationException: Ya hay un DataReader abierto asociado a este Command, debe cerrarlo primero.
   en System.Data.SqlClient.SqlInternalConnectionTds.ValidateConnectionForExecute(SqlCommand command)
   en System.Data.SqlClient.SqlConnection.ValidateConnectionForExecute(String method, SqlCommand command)
   en System.Data.SqlClient.SqlCommand.ValidateCommand(String method, Boolean async)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   en System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   en System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   en System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   en System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
------------------------------------------------------------
Log Entry : 10:13:24, miércoles, 7 de junio de 2017 :: From GenericRep7 :: The model backing the 'GaleriaContext' context has changed since the database was created. Consider using Code First Migrations to update the database (http://go.microsoft.com/fwlink/?LinkId=238269).
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 10:13:28, miércoles, 7 de junio de 2017 :: From A_Log2 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 10:48:03, domingo, 11 de junio de 2017 :: From GenericRep6 :: The model backing the 'GaleriaContext' context has changed since the database was created. Consider using Code First Migrations to update the database (http://go.microsoft.com/fwlink/?LinkId=238269).
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 8:28:00, martes, 13 de junio de 2017 :: From Loaders2 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 14:47:52, martes, 13 de junio de 2017 :: From GenericRep2 :: Attaching an entity of type 'Galeria.Model.Translation.ArtworkTranslations' failed because another entity of the same type already has the same primary key value. This can happen when using the 'Attach' method or setting the state of an entity to 'Unchanged' or 'Modified' if any entities in the graph have conflicting key values. This may be because some entities are new and have not yet received database-generated key values. In this case use the 'Add' method or the 'Added' entity state to track the graph and then set the state of non-new entities to 'Unchanged' or 'Modified' as appropriate.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 14:50:46, martes, 13 de junio de 2017 :: From GenericRep2 :: Attaching an entity of type 'Galeria.Model.Translation.AuthorTranslations' failed because another entity of the same type already has the same primary key value. This can happen when using the 'Attach' method or setting the state of an entity to 'Unchanged' or 'Modified' if any entities in the graph have conflicting key values. This may be because some entities are new and have not yet received database-generated key values. In this case use the 'Add' method or the 'Added' entity state to track the graph and then set the state of non-new entities to 'Unchanged' or 'Modified' as appropriate.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 8:57:02, miércoles, 14 de junio de 2017 :: From GenericRep2 :: Attaching an entity of type 'Galeria.Model.Translation.NationalityTranslations' failed because another entity of the same type already has the same primary key value. This can happen when using the 'Attach' method or setting the state of an entity to 'Unchanged' or 'Modified' if any entities in the graph have conflicting key values. This may be because some entities are new and have not yet received database-generated key values. In this case use the 'Add' method or the 'Added' entity state to track the graph and then set the state of non-new entities to 'Unchanged' or 'Modified' as appropriate.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 22:57:04, jueves, 15 de junio de 2017 :: From GenericRep3 :: Store update, insert, or delete statement affected an unexpected number of rows (0). Entities may have been modified or deleted since entities were loaded. See http://go.microsoft.com/fwlink/?LinkId=472540 for information on understanding and handling optimistic concurrency exceptions.
 :Inner Exception: System.Data.Entity.Core.OptimisticConcurrencyException: Store update, insert, or delete statement affected an unexpected number of rows (0). Entities may have been modified or deleted since entities were loaded. See http://go.microsoft.com/fwlink/?LinkId=472540 for information on understanding and handling optimistic concurrency exceptions.
   en System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.ValidateRowsAffected(Int64 rowsAffected, UpdateCommand source)
   en System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.Update()
   en System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.<Update>b__2(UpdateTranslator ut)
   en System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.Update[T](T noChangesResult, Func`2 updateFunction)
   en System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.Update()
   en System.Data.Entity.Core.Objects.ObjectContext.<SaveChangesToStore>b__35()
   en System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   en System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   en System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   en System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   en System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   en System.Data.Entity.Internal.InternalContext.SaveChanges()
------------------------------------------------------------
Log Entry : 9:07:53, lunes, 19 de junio de 2017 :: From MngTrans2 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 9:08:06, lunes, 19 de junio de 2017 :: From MngTrans3 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:41:05, lunes, 19 de junio de 2017 :: From MngNats2 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:01:30, martes, 20 de junio de 2017 :: From GenericRep7 :: Unable to create a constant value of type 'Galeria.Model.User'. Only primitive types or enumeration types are supported in this context.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:01:31, martes, 20 de junio de 2017 :: From GenericRep7 :: Unable to create a constant value of type 'Galeria.Model.User'. Only primitive types or enumeration types are supported in this context.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:01:32, martes, 20 de junio de 2017 :: From GenericRep7 :: Unable to create a constant value of type 'Galeria.Model.User'. Only primitive types or enumeration types are supported in this context.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:03:45, martes, 20 de junio de 2017 :: From GenericRep7 :: Unable to create a constant value of type 'Galeria.Model.User'. Only primitive types or enumeration types are supported in this context.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:04:59, martes, 20 de junio de 2017 :: From GenericRep7 :: Unable to create a constant value of type 'Galeria.Model.User'. Only primitive types or enumeration types are supported in this context.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:07:20, martes, 20 de junio de 2017 :: From GenericRep7 :: Unable to create a constant value of type 'Galeria.Model.User'. Only primitive types or enumeration types are supported in this context.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:55:07, martes, 20 de junio de 2017 :: From GenericRep7 :: Unable to create a constant value of type 'Galeria.Model.User'. Only primitive types or enumeration types are supported in this context.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 18:57:29, martes, 20 de junio de 2017 :: From GenericRep7 :: Unable to create a constant value of type 'Galeria.Model.User'. Only primitive types or enumeration types are supported in this context.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 21:43:56, martes, 20 de junio de 2017 :: From A_Log2 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 21:44:06, martes, 20 de junio de 2017 :: From A_Log2 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 21:45:47, martes, 20 de junio de 2017 :: From A_Log2 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
Log Entry : 21:55:41, martes, 20 de junio de 2017 :: From A_Log2 :: Referencia a objeto no establecida como instancia de un objeto.
 :Inner Exception: 
------------------------------------------------------------
